docs/samples/v1beta1/transformer/collocation/collocation.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: custom-transformer-collocation
+spec:
+  predictor:
+    containers:
+      - name: kserve-container
+        image: kserve/custom-model-grpc:latest
+        args:
+          - --model_name=custom-model
+          - --protocol=grpc-v2
```

Comment:
We do not need this argument for predictor, it is only for specifying the protocol between transformer and predictor

Reply:
done